scripts/src/java/org/oppia/android/scripts/xml/XmlSyntaxErrorHandler.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.android.scripts.xml
+
+import org.xml.sax.ErrorHandler
+import org.xml.sax.SAXParseException
+
+/**
+ * Class for custom error handling of the parse exceptions thrown by the parser. It collects all
+ * the syntax errors thrown by the parser, which can then later be retrieved.
```

Comment:
Could be made a bit more concise I think:

```suggestion
 * Custom XML parser error handler which collects syntax errors as they occur for later processing.
```

Reply:
Done